templates/kubernetes.html

```diff
@@ -0,0 +1,570 @@
+              </div>
+            </div>
+            </li>
+          <li class="p-matrix__item">
+            <div class="p-matrix__content">
+              <h3 class="p-matrix__title p-heading--four">拡張可能</h3>
+              <p class="p-matrix__desc">標準オープンソースのログ記録、モニタリング、アラート、ストレージ、ネットワーキング、コンテナランタイムを容易に統合。詳細なカスタマイズのためのコンサルティングを提供しています</p>
+            </div>
+          </li>
+          <li class="p-matrix__item">
+            <div class="p-matrix__content">
+              <h3 class="p-matrix__title p-heading--four">マネージドKubernetes</h3>
+              <div class="p-matrix__desc">
+                <p>リクエストに応じて、Canonicalのリモート運用部門がお客様の代わりにオンプレミスまたはお客様の選択したパブリッククラウドでKubernetesを構築および運用します。お客様の運用部門へのクラスター管理の引き渡しは、必要な時にいつでも行っていただけます</p>
+                <p><a href="#managed-packages">マネージドKubernetesに関する詳細&nbsp;&rsaquo;</a></p>
+              </div>
+            </div>
+          </li>
+          <li class="p-matrix__item">
+            <div class="p-matrix__content">
+              <h3 class="p-matrix__title p-heading--four">GPUとCUDAに対応</h3>
+              <p class="p-matrix__desc">K8sでの機械学習、AI、トランスコーディングのワークロードを加速するため、GPGPUリソースを自動的に検出し、構成します</p>
+            </div>
+          </li>
+          <li class="p-matrix__item">
+            <div class="p-matrix__content">
+              <h3 class="p-matrix__title p-heading--p-heading--four">コンバージド</h3>
+              <p class="p-matrix__desc">ストレージ、ネットワーキング、コンピュートは当社のハイパーコンバージドリファレンスアーキテクチャにオーバーレイされ、リソースの効率的な使用と高可用性フェイルオーバーが確保されます</p>
+            </div>
+            </li>
+          <li class="p-matrix__item">
+            <div class="p-matrix__content">
+              <h3 class="p-matrix__title p-heading--four">準拠</h3>
+              <p class="p-matrix__desc">Charmed Kubernetesはこれまで一貫して、K8sテストスイートのパフォーマンスとAPI仕様準拠の分野をリードしてきました</p>
+            </div>
+          </li>
+
+          <li class="p-matrix__item">
+            <div class="p-matrix__content">
+              <h3 class="p-matrix__title p-heading--four">回復力</h3>
+              <p class="p-matrix__desc">Charmed K8sのデプロイは高可用性の設計になっており、ダウンタイムなしでアップグレード可能です</p>
+            </div>
+          </li>
+          <li class="p-matrix__item">
+            <div class="p-matrix__content">
+              <h3 class="p-matrix__title p-heading--four">ベアメタル</h3>
+              <p class="p-matrix__desc">ベアメタルサーバー上でデプロイすることで、高可用性、GPGPUの自動検出、ソフトウェア定義ストレージ（SDS）を持った最高のパフォーマンスが可能になります</p>
+            </div>
+          </li>
+          <li class="p-matrix__item">
+            <div class="p-matrix__content">
+              <h3 class="p-matrix__title p-heading--four">ネイティブ</h3>
+              <p class="p-matrix__desc">ストレージ、アイデンティティ、その他のサービスを最適化するために、下層のパブリッククラウドSaaS（Software as a Service）との統合が可能です</p>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip--light is-bordered" id="bare-metal">
+    <div class="row">
+      <div class="col-8">
```

Comment:
Maybe remove this div to stop the single-character wrapping in the heading

Reply:
@anthonydillon Can you try removing the col-8? I think that it looks so much better.